admin/admin.php

```diff
@@ -1091,6 +1091,9 @@ function alm_save_repeater(){
 
       // Write Repeater Template
 	   try {
+			if ((defined('DISALLOW_FILE_EDIT') && DISALLOW_FILE_EDIT) || defined('DISALLOW_FILE_MODS') && DISALLOW_FILE_MODS){
```

Comment:
I'm considering this... I may want to ignore this as it could open up more issues than preventing issues.